python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/management_security.schema.yml

```diff
@@ -115,6 +115,21 @@ keys:
               description: |
                 cipher_list syntax follows the openssl cipher strings format.
                 Colon (:) separated list of allowed ciphers as a string.
+                If using EOS version 4.32.0F or later, use the setting `ciphers` instead of `cipher_list`.
```

Comment:
Maybe say "Not supported on EOS version starting 4.32.0F, use the `ciphers` setting instead."

Reply:
Fixed